tests/unit/views/ont/ONTSampleIndex.spec.js

```diff
@@ -5,6 +5,12 @@ import OntRequestFactory from '@tests/factories/OntRequestFactory.js'
 
 const ontRequestFactory = OntRequestFactory()
 
+vi.mock('@/composables/useLocationFetcher', () => ({
+  default: () => ({
+    fetchLocations: vi.fn().mockResolvedValue([]),
+  }),
+}))
+
```

Comment:
How does it work in PacbioSampleIndex without this mocking?

Reply:
The tests pass It shows a warning as it throws an error. Not sure why only this one is failing but looks like it changed when I changed from axios to fetch so suspect something is hidden which was exposed when I fixed the tests.